react-web/src/components/PointingSession.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
```

Comment:
Once https://github.com/maleck13/storypoint/pull/16 lands can this can `PointingSession` be removed?

Reply:
that can be removed once that PR lands :+1: